Sources/web3swift/Utils/Hooks/NonceMiddleware.swift

```diff
@@ -40,65 +32,9 @@ extension Web3.Utils {
                     self.nonceLookups[key] = value
                     i = i + 1
                 }
-
             }
         }
 
-        public init() {
-
-        }
-
-        // FIXME: Rewrite this to CodableTransaction
-//        func preAssemblyFunction(tx: inout CodableTransaction, contract: EthereumContract) -> (CodableTransaction, EthereumContract, Bool) {
-//            guard let from = tx.from else {
-//                // do nothing
-//                return (tx, contract, true)
-//            }
-//            guard let knownNonce = self.nonceLookups[from] else {
-//                return (tx, contract, true)
-//            }
-//
-//            let newNonce = knownNonce + 1
-//
-//            self.queue.async {
-//                self.nonceLookups[from] = newNonce
-//            }
-//
-//            // FIXME:
-//            tx.noncePolicy = .exact(newNonce)
-//            return (tx, contract, true)
-//        }
-
-//        func postSubmissionFunction(result: TransactionSendingResult) {
-//            guard let from = result.transaction.sender else {
-//                // do nothing
-//                return
-//            }
-//
-//            let newNonce = result.transaction.nonceRe
-//
-//            if let knownNonce = self.nonceLookups[from] {
-//                if knownNonce != newNonce {
-//                    self.queue.async {
-//                        self.nonceLookups[from] = newNonce
-//                    }
-//                }
-//                return
-//            }
-//            self.queue.async {
-//                self.nonceLookups[from] = newNonce
-//            }
-//            return
-//        }
-
-//        public func attach(_ web3: web3) {
-//            self.web3 = web3
-//            web3.eventLoop.monitoredUserFunctions.append(self)
-//            let preHook = AssemblyHook(function: self.preAssemblyFunction)
-//            web3.preAssemblyHooks.append(preHook)
-//            let postHook = SubmissionResultHook(function: self.postSubmissionFunction)
-//            web3.postSubmissionHooks.append(postHook)
-//        }
-
+        public init() { }
```

Comment:
Please move it to the line 18 so that it's after variables and before `functionToRun()`.